### DISCUSSION TITLE
[Design] New Antag Gamemode: Pirates

### DISCUSSION THREAD

**Discussion by OctusGit** [ParadiseSS13/Paradise#19482] [Documents]
### Why pirates?
As a person who has played multiple codebases throughout my time on SS13, I've found one game mode I enjoyed immensely. That would fit well in Paradise's atmosphere and as an action-packed situation for those facing Pirates.

Pirates would be a midround roll antag that has a chance to spawn like any other midround. The goal of pirates is to be more of a siege-focused game mode instead of an infestation-based (Xenos, Terrors, Blob) one. 

It would focus primarily on trying to disrupt workflow on the station while also giving players multiple directions on how to face a threat. Does security assault the ship with assistance to overwhelm the pirates? Does science make a bomb to hit the pirates, forcing them to abandon the ship? Or does everyone live with the fact they exist and try to survive? 

That depends on the context and opinion of the crew and round. But it already adds drama and conflict to the game, which is what all antags should accomplish.

With the current climate of midround antags, some variety is essential to keep the game fresh and exciting. There are a few things this game mode can add that will stand out from other mid-rounds. But as for now, here are a few ideas for what pirates can damage the crew.

### List of possible things pirates could do
- Pirates can siphon from Cargo and RnD. Blocking them from using their devices and siphoning cargo's budget over time if not dealt with.
- A power sink on the pirates ship that can steal the station's energy. The pirates fail only when it is destroyed, and power will return.
- Kidnapping of crew members via ransom like the contractor system. Deducting points from cargo or causing science to lose a tech level in a single category to pay back the cost.
- Raiding the vault for valuable items like the nuke core or champion belt.
- General stealing of items from the station, all departments have things of value, and they can easily be turned for a profit with the right connections.

### Possible Factions
For pirates, I would do the TG route of things and have a random choice of selectable species that can be midround. This would add variety and give them their unique personality in antag. Currently, there are four species I have in mind.

- Human
- Unathi
- Skeleton
- Vox

Each would have its unique style of ship and weapons they spawn with. I only have a little of what they would have, but that is a future work in progress. As for weapon sets, it is not set in stone.

[This document is a WIP, I will come back and update it whenever.

**Comment by McRamon** [comment]
I always imagined a drask pirate gang. Skeleton crew also feels fun, could use actual medieval pirate aesthetics, sort of a wizard themed pirate crew (if only not oldcrit).
 In general, pirates could work really well as midround costco nukies
